baybe/recommenders/bayesian.py

```diff
@@ -48,6 +53,11 @@ class BayesianRecommender(Recommender, ABC):
     ] = field(default="qEI")
     """The used acquisition function class."""
 
+    _acquisition_function: Optional[AcquisitionFunction] = field(
```

Comment:
I think you already explained at one point, but could you remind me why this change is necessary?

Reply:
please see see Azure work item 15200
this unifies the recommender signatures which were different for bayesian and other recommenders